src/mdbExtensionController.ts

```diff
@@ -210,6 +213,50 @@ export default class MDBExtensionController implements vscode.Disposable {
     }
   }
 
+  registerUriHandler = (): void => {
+    vscode.window.registerUriHandler({
+      handleUri: this._handleDeepLink,
+    });
+  };
+
+  _handleDeepLink = async (uri: vscode.Uri): Promise<void> => {
+    let command = uri.path.replace(/^\//, '');
+    if (!command.startsWith('mdb.')) {
+      command = `mdb.${command}`;
+    }
+
+    const parameters = queryString.parse(uri.query, {
```

Comment:
I guess the value in this as opposed to URLSearchParams is its parsing? Seems reasonable if so.

Reply:
Yeah - it converts booleans and numbers to the correct type rather than representing them as strings.